test/bdd/fixtures/testdata/presentationdefinitions.json

```diff
@@ -1,9 +1,63 @@
 {
-  "CreditCardStatement": {
+  "credit_card_stmt:remote": {
```

Comment:
wrong input descriptor format,

supposed to `"id": "credit_card_stmt:remote",`

Reply:
@sudeshrshetty this new configuration format has been in use since #234

Reply:
ok, if you aren't sending it as it is to wallet then that's ok